specification/cognitiveservices/OpenAI.Inference/models/images.tsp

```diff
@@ -0,0 +1,137 @@
+import "@azure-tools/typespec-azure-core";
+import "@typespec/http";
+
+using Azure.Core;
+using TypeSpec.Http;
+
+namespace Azure.OpenAI;
+
+@doc("The desired size of the generated images. Must be one of 256x256, 512x512, or 1024x1024.")
+enum ImageSize {
+  @doc("Small image size (256x256).")
+  Small: "256x256",
+
+  @doc("Medium image size (512x512).")
+  Medium: "512x512",
+
+  @doc("Large image size (1024x1024).")
+  Large: "1024x1024",
+}
+
+@doc("Represents the request data used to generate images.")
+model ImageRequest {
+  @doc("A description of the desired images.")
+  prompt: string;
+
+  @doc("The number of images to generate (defaults to 1).")
+  n?: int32;
```

Comment:
We should provide the default value if there is one.

Reply:
is there an annotation to do that? in this case the default is implicit, if not present, the service will use `1`

Reply:
Paraphrasing: it's a service-inferred default rather than a client-coded one. The longer version of this might be something like "If not provided, the image generation request operation will behave as if n=1 and generate a single image."

Reply:
n?: int32 = 1;